tasks/main.yml

```diff
@@ -10,10 +10,11 @@
 
 # tasks for apache
 - name: ensure apache packages are installed
-  yum: name={{ item }} state=present lock_timeout=180
-  with_items:
-    - httpd
-    - collectd-apache
+  yum: name="{{ packages }}" state=present lock_timeout=180
+  vars:
+    packages:
+      - httpd
+      - collectd-apache
```

Comment:
🆒

Reply:
This syntax is deprecated and will throw warnings. Use something like:

`yum: name="{{ ['httpd','collectd-apache'] }}"`